doc/http-api/backend_swagger.yml

```diff
@@ -187,14 +175,14 @@ paths:
                 description: the recipient of the message
                 type: string
                 format: JID
-              msg:
+              body:
                 description: the message text
                 type: string
       description: Send a chat message from one user to another.
       responses:
         200:
           description: Message was sent
-  /messages/{caller}/{other}/{limit}:
+  /messages/{caller}/{other}:
```

Comment:
Also, it should be possible to fetch the whole archive of a user. In other words the `other` parameter should be optional.